builtin/mainmenu/dlg_create_world.lua

```diff
@@ -51,46 +159,173 @@ local function create_world_formspec(dialogdata)
 				end
 			end
 		end
+
+		local ds = (gameconfig:get("disallowed_settings") or ""):split()
```

Comment:
The game setting is better called 'disallowed_mapgen_flags', because that is what it is. 'settings' can mean almost anything, and in code means all settings.

Reply:
It's literally settings. All settings.

Reply:
Yes that is what i mean, settings can mean any setting.
The usage of the feature is to disallow certain mapgen flags, so that is how it should be named, otherwise someone may think it can be used to disallow something that is not a mapgen flag.
@rubenwardy what is your objection to my suggestion?

Reply:
The documentation of this does not make it clear this is for mapgen flags only.

Anyway, from discussion:
> The name disallowed_settings is intentionally generic. With disallowed_settings, the feature can later be trivially extended to include other settings that are not mapgen related

I am not sure it is acceptable to pile many different types of setting into one string. It may be better to keep this for mapgen flags only and add more configs if required.